### PR TITLE
EWPP-3827: Drop 9.4 core support and update entity_browser patch and selenium.

### PR DESCRIPTION
**Pull request by nagyad**

### CODE REVIEW

**Review by 22Alexandra — DISMISSED** [review]

**Comment by 22Alexandra** [comment]
We should update the info files and docker php image to 8.2 within this PR too, otherwise we might forget.

**Review by 22Alexandra — DISMISSED** [review]

File: phpunit.xml.dist [hunk 1]
```diff
@@ -7,7 +7,7 @@
     <env name="SIMPLETEST_BASE_URL" value="${drupal.base_url}"/>
     <env name="SIMPLETEST_DB" value="mysql://${drupal.database.user}:${drupal.database.password}@${drupal.database.host}:${drupal.database.port}/${drupal.database.name}"/>
     <env name="SIMPLETEST_SPARQL_DB" value="sparql://${drupal.sparql.host}:${drupal.sparql.port}/?module=sparql_entity_storage"/>
-    <env name="MINK_DRIVER_ARGS_WEBDRIVER" value='["${selenium.browser}", null, "${selenium.host}:${selenium.port}/wd/hub"]'/>
+    <env name="MINK_DRIVER_ARGS_WEBDRIVER" value='["${selenium.browser}", {"browserName":"chrome","chromeOptions":{"w3c": false, "args":["--no-sandbox", "--start-maximized", "--disable-gpu", "--window-size=1440,900", "--disable-dev-shm-usage", "--disable-setuid-sandbox", "--disable-web-security", "--DNS-prefetch-disable", "--disable-translate", "--ignore-certificate-errors", "--test-type", "--disable-extensions", "--incognito", "--disable-infobars"]}}, "${selenium.host}:${selenium.port}/wd/hub"]'/>
```
Comment (brummbar) [inline phpunit.xml.dist]: Window size here should reflect the old one and be 1280x800.

Comment (brummbar) [reply phpunit.xml.dist]: We also need to bring these to behat.yml.dist .
I'm copying here the ones from joinup, update them based on the ones used above:
```
capabilities:
          browser: chrome
          nativeEvents: true
          marionette: true
          browserName: chrome
          version: '*'
          extra_capabilities:
            chromeOptions:
              w3c: false
              args:
                - '--no-sandbox'
                - '--start-maximized'
                - '--disable-gpu'
                - '--window-size=1600,1200'
                - '--disable-dev-shm-usage'
                - '--disable-setuid-sandbox'
                - '--disable-web-security'
                - '--DNS-prefetch-disable'
                - '--disable-translate'
                - '--ignore-certificate-errors'
                - '--test-type'
                - '--disable-extensions'
                - '--incognito'
                - '--disable-infobars'
                - '--allowed-ips=*'
                - '--whitelisted-ips=*

File: .drone.yml [hunk 2]
```diff
@@ -109,17 +109,17 @@ pipeline:
...
     - { PHP_VERSION: 8.1, CORE_VERSION: 10.0, TYPE: behat, BATCH: '@batch1' }
     - { PHP_VERSION: 8.1, CORE_VERSION: 10.0, TYPE: behat, BATCH: '@batch2' }
     - { PHP_VERSION: 8.1, CORE_VERSION: 10.0, TYPE: behat, BATCH: '@batch3' }
+
+    - { PHP_VERSION: 8.2, CORE_VERSION: 10.0, TYPE: phpunit }
+    - { PHP_VERSION: 8.2, CORE_VERSION: 10.0, TYPE: behat, BATCH: '@batch1' }
+    - { PHP_VERSION: 8.2, CORE_VERSION: 10.0, TYPE: behat, BATCH: '@batch2' }
+    - { PHP_VERSION: 8.2, CORE_VERSION: 10.0, TYPE: behat, BATCH: '@batch3' }
+
+    - { PHP_VERSION: 8.2, CORE_VERSION: 10.1, TYPE: phpunit }
```
Comment (brummbar) [inline .drone.yml:127]: we should add also 8.1 and 10.1. Before the matrix had only 3 entries as 10.x was not compatible with 8.0.

**Review by brummbar — CHANGES_REQUESTED** [review]

**Review by brummbar — APPROVED** [review]

**Review by 22Alexandra — APPROVED** [review]

**Pull request merged**